src/Microsoft.Health.Fhir.SqlServer/Features/Search/Expressions/Visitors/QueryGenerators/SqlQueryGenerator.cs

```diff
@@ -485,13 +485,31 @@ public object VisitTable(TableExpression tableExpression, SearchOptions context)
                     _includeCtes.Add(TableExpressionName(_tableExpressionCounter));
                     break;
                 case TableExpressionKind.IncludeUnionAll:
-                    StringBuilder.AppendLine("SELECT Sid1, IsMatch, IsPartial ");
+                    StringBuilder.Append("SELECT Sid1, IsMatch, IsPartial ");
+                    if (context.SupportedSortParamExists())
```

Comment:
>context.SupportedSortParamExists() [](start = 24, length = 34)

nit: I would use bool variable to store result of this function, it looks a bit heavy for me and it result can't be change inside current context, so why not calculate it once?